ExamenProeveUnity/Assets/Scripts/Runtime/Products/Box.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Runtime.Products
+{
+    public class Box : MonoBehaviour
+    {
+        [SerializeField] private ProductScriptableObject content;
+        [SerializeField] private int itemCount = 1;
+
+        [SerializeField] private List<Image> images;
+        [SerializeField] private List<TMP_Text> countTexts;
+
+        private Grabbable _grabbable;
+
+        private void Awake()
+        {
+            gameObject.TryGetComponent<Grabbable>(out _grabbable);
+            gameObject.GetComponent<Interactable>().onInteractionClicked.AddListener(OpenBox);
+            
+            if (content == null) return;
```

Comment:
ik zie dat content er MOET zijn dus misschien als het er niet is een warning printen en het script mogelijk uitzetten?